vnext/Shared/Networking/RedirectHttpFilter.cpp

```diff
@@ -50,10 +52,14 @@ RedirectHttpFilter::RedirectHttpFilter(
   if (auto baseFilter = m_innerFilter.try_as<IHttpBaseProtocolFilter>()) {
     baseFilter.AllowAutoRedirect(false);
     baseFilter.AllowUI(false);
+    baseFilter.CacheControl().ReadBehavior(HttpCacheReadBehavior::NoCache);
+    baseFilter.CacheControl().WriteBehavior(HttpCacheWriteBehavior::NoCache);
   }
   if (auto baseFilter = m_innerFilterWithNoCredentials.try_as<IHttpBaseProtocolFilter>()) {
     baseFilter.AllowAutoRedirect(false);
```

Comment:
RN on Android by default follows browser behavior, and does allow redirects.

Reply:
See class `RedirectHttpFilter`.
Redirects don't work on some cases deemed as "irregular redirects" by the underlying HTTP stack (WinInet), irrecoverably throwing `ERROR_HTTP_REDIRECT_NEEDS_CONFIRMATION` when invoked from the WinRT layer.

Redirects are still followed, only manually.